@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -577,6 +577,12 @@ export class TextElementsRenderer {
         return memoryUsage;
     }
 
+    prepopulateScreen(boxes: Math2D.Box[]) {
+        for (const box of boxes) {
+            this.m_screenCollisions.allocate(box);
```

Comment:
I wonder how we handle priorities for the screen collisions, but I guess that's not relevant for your ticket.

Reply:
I will add a comment.